docs/walkthrough/walkthrough_data.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Walkthrough data
+summary: Example data for walkthrough and testing
+permalink: walkthrough_data.html
+datatable: true
+---
+{% assign walkthrough_folder = "walkthrough/data/" %}
+
+This page describes the dataset that is used in the 
+[Walkthrough](walkthrough.html) for testing _ReadTools_ binaries.
+A list with all the files and checksum could be found 
+[here]({{walkthrough_folder}}/checksum).
+
+The data is a modified subset of reads from 
+[SRR1931701](https://www.ncbi.nlm.nih.gov/sra/?term=SRR1931701).
+Concretely, the first 100 read-pairs were downloaded. Out of this 100 
+read-pairs, 3 were modified by masking read-ends (MASKED reads). 
+Additionally,  the quality of 1 was changed (LOWQUAL).
+These 103 read-pairs were encoded in the different formats 
+mentioned below ([Reads data summary](#reads-data-summary)),
+with the following additions/changes:
+
+* Single end: only the first read of each pair was kept and marked as 
+  unpaired following the specifications of each format.
+* Quality encoding: a subset of files were duplicated and converted
+  to the Illumina encoding for testing quality detection/correction.
+* Read name: a subset of files were duplicated and read names were 
+  changed according to different conventions (Illumina-legacy and 
+  Casava) for testing lossless conversion and barcode handling.
+* Barcodes: single or dual barcoding indexes based on a default set of
+  barcodes ([Other data](#other-data)) were added to the files for 
+  testing lossless conversion and de-multiplexing.
+
+For both unmapped and mapped reads, the sort order is _unsorted_ but 
+they are grouped by queryname (not reflected in the header). Mapped
+reads were generated by [bwa]({{site.data.software.bwa}}).
```

Comment:
In principle this is in the header of the SAM/BAM/CRAM file, and it is not important for the current dataset. But true, algorithm (mem) and version is important. It was just mapped with default parameters, so I added a short sentence to point that out: "Mapped reads were generated by bwa mem version 0.7.12 and default parameters"